source/retryable-reads/retryable-reads.rst

```diff
@@ -51,22 +51,23 @@ An error is considered retryable if it meets any of the following criteria:
 
 - a server error response with any the following codes:
 
-=============================== ==============
-**Error Name**                  **Error Code**
-=============================== ==============
-ExceededTimeLimit               262
-InterruptedAtShutdown           11600
-InterruptedDueToReplStateChange 11602
-NotWritablePrimary              10107
-NotPrimaryNoSecondaryOk         13435
-NotPrimaryOrSecondary           13436
-PrimarySteppedDown              189
-ShutdownInProgress              91
-HostNotFound                    7
-HostUnreachable                 6
-NetworkTimeout                  89
-SocketException                 9001
-=============================== ==============
+================================== ==============
+**Error Name**                     **Error Code**
+================================== ==============
+ExceededTimeLimit                  262
+InterruptedAtShutdown              11600
+InterruptedDueToReplStateChange    11602
+NotWritablePrimary                 10107
+NotPrimaryNoSecondaryOk            13435
+NotPrimaryOrSecondary              13436
+PrimarySteppedDown                 189
+ReadConcernMajorityNotAvailableYet 134
```

Comment:
Note: The error named `ReadConcernMajorityNotAvailableYet` is longer than any existing errors in this list, so I needed to resize the table to fit it in.

Reply:
In these cases the "Hide whitespace" changes checkbox is really useful.